src/e2e/tatum.token.spec.ts

```diff
@@ -0,0 +1,280 @@
+import { Ethereum } from '../dto'
+import { Network, TatumSDK } from '../service'
+
+describe('Tatum token', () => {
+  let tatum: TatumSDK<Ethereum>
+  beforeAll(async () => {
+    tatum = await TatumSDK.init<Ethereum>({
+      network: Network.ETHEREUM_SEPOLIA,
```

Comment:
For future we can think about some abstraction for all networks. But now this is ok.

Reply:
With the e2e tests it would be quite difficult to cover all of them